apps/voting-disputable/contracts/DisputableVoting.sol

```diff
@@ -733,160 +759,169 @@ contract DisputableVoting is IForwarderWithContext, DisputableAragonApp {
+    * @param _setting Setting instance applicable to the vote
+    * @return True if the vote is within its quiet ending period
     */
     function _withinQuietEndingPeriod(Vote storage _vote, Setting storage _setting) internal view returns (bool) {
         return getTimestamp64() >= _durationStartDate(_vote, _setting.quietEndingPeriod);
     }
 
     /**
-    * @dev Internal function to check if a vote is within its execution delay window
-    *      It assumes the pointer to the vote is valid
-    * @return True if the given vote is within its execution delay window
+    * @dev Tell if a vote is within its execution delay window
+    * @param _vote Vote instance being queried
+    * @param _setting Setting instance applicable to the vote
+    * @return True if the vote is within its execution delay window
     */
     function _withinExecutionDelayWindow(Vote storage _vote, Setting storage _setting) internal view returns (bool) {
         return getTimestamp64() < _finalVoteEndDate(_vote).add(_setting.executionDelay);
     }
 
     /**
-    * @dev Internal function to calculate the original end date of a vote
-    *      It does not consider the paused nor quiet ending extensions
-    *      It assumes the pointer to the vote is valid
+    * @dev Calculate the original end date of a vote
+    *      It does not consider extensions from pauses or the quiet ending mechanism
+    * @param _vote Vote instance being queried
+    * @return Datetime of the vote's original end date
     */
     function _originalVoteEndDate(Vote storage _vote) internal view returns (uint64) {
         return _vote.startDate.add(voteTime);
     }
 
     /**
-    * @dev Internal function to calculate the end date of a vote
-    *      It considers the extensions due to quiet ending.
-    *      The pause duration will be included only after the vote has "returned" from being paused
-    *      It assumes the pointer to the vote is valid
+    * @dev Calculate the end date of a vote.
+    *      It considers extensions from pauses and the quiet ending mechanism.
+    *      The pause duration will only be included after the vote has "resumed" from its pause, as we do not know how long the pause will be in advance.
+    * @param _vote Vote instance being queried
+    * @return Datetime of the vote's end date
     */
     function _finalVoteEndDate(Vote storage _vote) internal view returns (uint64) {
         uint64 endDateAfterPause = _originalVoteEndDate(_vote).add(_vote.pauseDuration);
         return endDateAfterPause.add(_vote.quietEndingExtendedSeconds);
     }
 
     /**
-    * @dev Internal function to tell whether a vote was flipped or not based on its snapshot support
-    *      It assumes the pointer to the vote is valid
+    * @dev Tell if a vote was flipped in its most recent quiet ending period
+    *      This function assumes that it will only be called after the most recent quiet ending period has already ended
```

Comment:
I thought this note was important to highlight to make sure I'm understanding how this is meant to be used.